packages/net-modules.spec

```diff
@@ -46,7 +45,7 @@ install %{SOURCE2} %{buildroot}%{_sysconfdir}/mesos-slave/
 
 ############################################
 %files
-%{_libdir}/mesos/libmesos_network_isolator*.so
+/usr/local/lib/mesos/libmesos_network_isolator*.so
```

Comment:
The RPM builder resolves `%{_libdir}` to `/usr/lib64/`, but net-modules compilation puts the files in `/usr/lib`. So you end up seeing this error:
```
RPM build errors:
File not found by glob: /root/rpmbuild/BUILDROOT/net-modules-0.2.0-1.custom.x86_64/usr/lib64/mesos/libmesos_network_isolator*.so
```

I'm not sure if its net-modules or the rpm builder that needs to be reconfigured. Any thoughts @karya0 ?